snapshot-labs/sx-monorepo
# feat: Explore page with snapshot spaces

### Summary
Closes: #297 
- Explore page support for snapshot spaces
- Remove favorite button on all space cards

### How to test
- Update `.env` with `VITE_ENABLED_NETWORKS=s,eth,oeth,matic,arb1,sn`
- Go to http://localhost:8080/#/explore
- You should see new protocol selection menu and spaces related to the protocol
